VirtoCommerce.OrderModule.Web/Module.cs

```diff
@@ -170,6 +172,11 @@ public override void PostInitialize()
             var allOrderKnownTypes = new[] { typeof(Shipment), typeof(PaymentIn), typeof(CustomerOrder) }.Concat(allPaymentMethodTypes).Concat(allShippingmethodTypes).ToArray();
             httpConfiguration.Formatters.XmlFormatter.SetSerializer<CustomerOrder>(new DataContractSerializer(typeof(CustomerOrder), allOrderKnownTypes));
             httpConfiguration.Formatters.XmlFormatter.SetSerializer<CustomerOrderSearchResult>(new DataContractSerializer(typeof(CustomerOrderSearchResult), allOrderKnownTypes));
+
+            //customer order workflow
+            AbstractTypeFactory<IOperation>.OverrideType<CustomerOrder, CustomerOrderWorkflow>();
+            AbstractTypeFactory<CustomerOrder>.OverrideType<CustomerOrder, CustomerOrderWorkflow>()
+                .WithFactory(() => new CustomerOrderWorkflow { OperationType = "CustomerOrder" });
```

Comment:
remove the factory method. Assign the OperationType inside CustomerOrderWorkflow constructor

Reply:
it's fixed